<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyQtClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendButton1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameLE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textLE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="messagesWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
